README.md

```diff
@@ -1,44 +1,57 @@
-NOTE: This is a WIP - I plan to write proper documentation and make a formal release soon.
+# graphql_client
 
-Derive Rust code to safely interact with queries written in the GraphQL query language.
+![CI Status](https://img.shields.io/travis/tomhoule/graphql-client.svg)
+![docs](https://docs.rs/mio/badge.svg)
+![crates.io](https://img.shields.io/crates/d/graphql-client.svg)
+![license](https://img.shields.io/github/license/mashape/apistatus.svg)
+![PRs Welcome](https://img.shields.io/badge/PRs-welcome-brightgreen.svg)
 
-This library does not provide any networking, caching or other client functionality, it is just meant to make it easy to interact with a GraphQL query and the corresponding response in a strongly typed way. Building a client can be as simple as this:
+Derive Rust code to safely and ergonomically manipulate GraphQL queries.
+
+This library does not provide any networking, caching or other client functionality yet, it is just meant to make it easy to interact with a GraphQL query and the corresponding response in a strongly typed way. Making a request can be as simple as this:
 
 ```rust
 #[derive(GraphQLQuery)]
 #[gql(
-    query = "/graphql/queries/my_query.graphql",
-    schema = "/graphql/schema.graphql"
+    query_path = "src/graphql/queries/my_query.graphql",
+    schema_path = "src/graphql/schema.json"
 )]
 struct MyQuery;
 
 fn perform_my_query(variables: &my_query::Variables) -> Result<(), failure::Error> {
     let body = MyQuery::expand(variables);
     let client = reqwest::Client::new();
-    let res: HttpResponse<graphql_client::Response<my_query::ResponseData>> = client.post("/graphql", body)?;
-    println!("{:#?}", res.body);
+    let mut res: HttpResponse<graphql_client::Response<my_query::ResponseData>> = client.post("/graphql").json(&body).send()?;
+    println!("{:#?}", res.text());
+    Ok(())
 }
 ```
 
+The paths in the `gql` attribute are relative to the directory where your `Cargo.toml` is located.
+
+The GraphQL schema language and schema.json are both supported as sources for the schema.
+
+`serde_derive` needs to be visible in the context of the `GraphQLQuery` derive (add it as an `extern crate`).
+
 ## Features
```

Comment:
I would put the whole features section a little bit more towards the start, maybe right after the recipe.